colossalai/pipeline/schedule/_utils.py

```diff
@@ -118,15 +169,18 @@ def merge_batch(data: List[Any]) -> Any:
     flattened_data = []
     tree_spec = None
     for d in data:
-        elems, tree_spec = tree_flatten(d)
+        print('d: ', d)
```

Comment:
Please remove this

Reply:
will push a new version after test is done!

Reply:
done!